mod-audit-server/src/main/java/org/folio/services/configuration/Setting.java

```diff
@@ -4,12 +4,14 @@
 
 @Getter
 public enum Setting {
+  AUTHORITY_RECORDS_PAGE_SIZE(SettingGroup.AUTHORITY, Setting.RECORDS_PAGE_SIZE_KEY),
+  INVENTORY_RECORDS_PAGE_SIZE(SettingGroup.INVENTORY, Setting.RECORDS_PAGE_SIZE_KEY),
+  MARC_RECORD_PAGE_SIZE(SettingGroup.MARC, Setting.RECORDS_PAGE_SIZE_KEY);
```

Comment:
You shouldn't create new setting for MARC, but use already existed settings for authority and inventory. 
There shouldn't be different configuration for instance and MARC bib records